src/mame/atari/atarigt.cpp

```diff
@@ -751,7 +751,19 @@ INPUT_PORTS_END
 static INPUT_PORTS_START( primrage )
 	PORT_INCLUDE( common )
 
-	/* primrage uses one of the 4 action buttons as start */
+	/* primrage has a dedicated start button */
+	PORT_MODIFY( "P1_P2" )
+	PORT_BIT( 0x00000001, IP_ACTIVE_LOW, IPT_UNUSED )
+	PORT_BIT( 0x00000002, IP_ACTIVE_LOW, IPT_BUTTON4 ) PORT_PLAYER(1)
+	PORT_BIT( 0x00000004, IP_ACTIVE_LOW, IPT_UNUSED )
+	PORT_BIT( 0x00000008, IP_ACTIVE_LOW, IPT_BUTTON4 ) PORT_PLAYER(2)
+	PORT_BIT( 0x000000F0, IP_ACTIVE_LOW, IPT_UNUSED )
+INPUT_PORTS_END
+
+static INPUT_PORTS_START( primrageo )
+	PORT_INCLUDE( common )
+
+	/* primrage20 & primrageo use one of the 4 action buttons as start */
```

Comment:
You don’t need the `IPT_UNUSED` here – it will fall through to what it inherits from `common` for the bits you don’t modify.  Also, please follow the style of the file and leave two blank lines between each set of port definitions.

Reply:
I think the IPT_UNUSED is necessary in this case because we are "breaking up" a byte-sized unused block in common.   I'm not super-familiar with the ioport code, but see: ioport.cpp#L41

`
// temporary: set this to 1 to enable the originally defined behavior that
// a field specified via PORT_MODIFY which intersects a previously-defined
// field completely wipes out the previous definition
#define INPUT_PORT_OVERRIDE_FULLY_NUKES_PREVIOUS    1
`

I have validated this behavior is happening, as well.  I originally didn't redefine the unused block, but it will trigger the undocumented debug menu in the game when if the developer flags have been set without it.  

The style wasn't clear to me--- There's only one line between common and tmek, and two between tmek and primrage, so I had inferred that maybe there was only 2 between different games.   I've added a 2nd line between primrage/primrageo.

Reply:
I thought that had been disabled ages ago – it was supposed to be temporary.  I hate “temporary” things that stick around forever.